tests/liquidity_provision/test_liquidity_provision.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from playwright.sync_api import Page, expect
+from vega_sim.service import VegaService
+from conftest import init_vega
+from fixtures.market import setup_continuous_market
+from actions.utils import next_epoch
+
+@pytest.fixture(scope="module")
+def vega(request):
+    with init_vega(request) as vega:
+        yield vega
+
+
+@pytest.fixture(scope="module")
+def continuous_market(vega):
+    return setup_continuous_market(vega)
+
+
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_liquidity_provision_amendment(continuous_market, vega: VegaService, page: Page):
+    # TODO Refactor asserting the grid
+    page.goto(f"/#/liquidity/{continuous_market}")
+    page.get_by_test_id("manage-vega-wallet").click()
+    page.locator('[role="menuitemradio"] >> .mr-2.uppercase').nth(1).click(position={ "x": 0, "y": 0}, force=True)
```

Comment:
what's this for? I always wince a bit when I see something clicked with co-ordinates. Is there a better way to do this?

Reply:
Playwright by default will click the middle of an element
annoyingly for us that means clicking the copy button so they it doesn't actually click what we want it to:
<img width="382" alt="image" src="https://github.com/vegaprotocol/console-test/assets/133853298/509aaf1a-d26d-479c-be31-136375516a1f">
Can fix this in the future by renaming the wallet from "mm" to "marketMaker" but I don't want to do that yet just incase it breaks loads of tests'
I'll add a comment explaining why and a TODO